src/libclipper/include/clipper/json_util.hpp

```diff
@@ -0,0 +1,362 @@
+#ifndef CLIPPER_LIB_JSON_UTIL_H
+#define CLIPPER_LIB_JSON_UTIL_H
+#include <rapidjson/document.h>
+#include <rapidjson/error/en.h>
+#include <rapidjson/stringbuffer.h>
+#include <rapidjson/writer.h>
+#include <clipper/datatypes.hpp>
+#include <stdexcept>
+
+using clipper::Input;
+using clipper::InputType;
+using clipper::Output;
+using clipper::OutputType;
+using clipper::VersionedModelId;
+using rapidjson::Type;
+
+static std::vector<std::string> kTypeNames = {
```

Comment:
The intended comment is listed below the vector declaration. Is the content of the comment clear enough? I can move the comment up and change it if necessary.